Remotion/Web/Development.WebTesting.ControlObjects/TabbedMenuControlObject.cs

```diff
@@ -165,8 +165,9 @@ private static IReadOnlyList<ItemDefinition> GetMenuItemOrSubMenuItemDefinitions
                                   itemScope[DiagnosticMetadataAttributes.ItemID],
                                   i + 1,
                                   itemScope.Text.Trim(),
-                                  itemScope[DiagnosticMetadataAttributes.IsDisabled] == "true"))
-                      .ToList());
+                                  itemScope[DiagnosticMetadataAttributes.IsDisabled] == "true",
+                                  itemScope.FindCss("a")?["accesskey"] ?? string.Empty))
+                  .ToList());
```

Comment:
```suggestion
                      .ToList());
```
Sorry, whitespace counting. my bad. Indent of ToList() was originally correct before thsi PR.